### DISCUSSION TITLE
What is non-adaptive customisation called?

### DISCUSSION THREAD

**Discussion by bittlingmayer** [machinetranslate/machinetranslate.org#460] [General]
What is the best name for the typical neural customisation like that of Google or Microsoft?

Just `fine-tuning`?  Should we have an article on this?

It seems that the customisation article and the adaptive article are missing that contrast.

**Comment by bittlingmayer** [comment]
This makes a lot of things clear: https://github.com/machinetranslate/machinetranslate.org/pull/461/files

Really it needs to be

```
{% assign customizable_apis = site.data.apis | where 'fine-tuning' | concat: site.data.apis | where 'glossary' | concat: site.data.apis | where 'formality' | concat: site.data.apis | where 'adaptive' %}
```

**Comment by cefoo** [comment]
Is `retraining` too far off?
